tests/e2e/provisioning/internal/hyperscaler/azure/client.go

```diff
@@ -0,0 +1,35 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/eventhub/mgmt/2017-04-01/eventhub"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-05-01/resources"
+)
+
+type AzureInterface interface {
```

Comment:
(nit) Azure is redundant, it reads `azure.AzureInterface`.

Reply:
changed to `Interface`